Added more details to README.md (#2)
Black-Boys-Code/TicTacToe: README.md

## Tic Tac Toe

- What is this project?

> This project was build for the black boys code organization. It's intetions is to be used for teaching fundamental ideas in building a website. 

Reviewer: ```suggestion
This project was build for the Black Boys Code organization. Its intentions is to be used for teaching fundamental ideas in building a website. 
```

- How does it work?

> Very simple. Open the index.html page in your browser. 

Reviewer: ```suggestion
Very simple, go to [](https://black-boys-code.github.io/TicTacToe). Or open this repo in Glitch project.
)```

- Who will use this project?

> Students learning the basics of web development. 

Reviewer: ```suggestion
Black Boys Code students learning the basics of web development. 
```
